cli/src/pcluster/models/cluster.py

```diff
@@ -310,6 +316,14 @@ def _validate_and_parse_config(self, suppress_validators, validation_failure_lev
 
         except ValidationError as e:
             # syntactic failure
+            # Process error message. Because there are some @pre_load(s) changing the structure of the schema,
+            # We need to modify the error message to be aligned with the structure before @pre_load(s)
+            error_massage = e.args[0]
+            if "Scheduling" in error_massage:
+                scheduling = error_massage["Scheduling"]
+                scheduler_settings = scheduling.get("Slurm") or scheduling.get("Awsbatch")
```

Comment:
temporarily let's define an enum for the available schedulers, in this way if we add another scheduler this code shouldn'ìt change.

Reply:
Done by reusing constant SUPPORTED_SCHEDULERS instead of creating a Enum